lib/decidim/internal_evaluation/version.rb

```diff
@@ -4,11 +4,11 @@ module Decidim
   # This holds the decidim-meetings version.
   module InternalEvaluation
     def self.version
-      "0.0.1"
+      "0.0.2"
     end
 
     def self.decidim_version
-      "~> 0.29.0"
+      "0.30.0"
```

Comment:
Don't use a fixed version here. Otherwise, it won't be compatible with 0.30.1

```suggestion
      "~> 0.30.0"
```